### PR TITLE
process: enable auto-approve for codegen via app

### PR DESCRIPTION
**Pull request by chingor13**
Enables the [auto-approve app](https://github.com/googleapis/repo-automation-bots/tree/main/packages/auto-approve)

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -4,5 +4,6 @@
 # For syntax help see:
 # https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners#codeowners-syntax
 
+.github/auto-approve.yml  @googleapis/github-automation
```
Comment (bshaffer) [inline .github/CODEOWNERS:7]: I can't find this one, should it be `@yoshi-automation` ?

**Review by bshaffer — CHANGES_REQUESTED** [review]

File: .github/CODEOWNERS (hunk 1 above)
Comment (bshaffer) [reply .github/CODEOWNERS:7]: I found it! It is a team, not a user. What role does it need to have?

Comment (chingor13) [reply .github/CODEOWNERS:7]: The team needs explicit write access

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -4,5 +4,6 @@
...
 
-*     @googleapis/yoshi-php yoshi-approver
+*                         @googleapis/yoshi-php yoshi-approver
```
[the remark this answers is not in the document]
Comment (chingor13) [reply .github/CODEOWNERS]: Fixed

File: .github/CODEOWNERS (hunk 1 above)
Comment (bshaffer) [reply .github/CODEOWNERS:7]: Done!

**Review by bshaffer — APPROVED** [review]

**Pull request merged**